packages/rosary-tauri/rosary-tauri.pacscript

```diff
@@ -0,0 +1,19 @@
+pkgname=rosary-tauri
```

Comment:
`.deb` based packages must be named `${pkgname}-deb`, as they are handled specially by pacstall. Also, please use quotes for all variables. You want this to look like:
```bash
pkgname='rosary-tauri-deb'
gives="${pkgname/-deb/}" # or gives='rosary-tauri'
```
the folder name, pacscript name, and name in `packagelist` should also be updated to `rosary-tauri-deb`

###### P.S. - it is highly recommended to take a look at https://github.com/pacstall/pacstall/wiki/Pacscript-101 to get familiar with our styling and slight differences from `PKGBUILD`.